src/values/utils.ts

```diff
@@ -29,6 +33,16 @@ export class StaticValue<P, T extends GroqType> {
         }
       })(this.data)
     }
+
+    if (this.data instanceof Map) {
+      return (function* (data) {
+        // This can be simplified when using es2015 target
+        for (const [id, value] of Array.from(data.entries())) {
+          yield fromJS(value)
+        }
```

Comment:
We're changing the target to `es6 and supports esm-modules` in https://github.com/sanity-io/groq-js/pull/79.
When you say simplify, is this what you have in mind:
```suggestion
        for (const [id, value] of data) {
          yield fromJS(value)
        }
```

Reply:
Yes, alternatively this as the `id` is unused in the loop:
```ts
for (const value of data.values()) {
```